service/routes.py

```diff
@@ -495,15 +485,12 @@ def get(self):
         """Returns all of the customers"""
         app.logger.info("Request for customer list")
         
-        all_query_key = ["username", "first_name", "last_name", "prefix_username"]
-        for key in request.args.keys():
-            if key not in all_query_key:
-                raise UnsupportedKeyError("The query key: '" + key + "' is not supported.")
```

Comment:
I changed this because we have a test with an unsupported query parameter for addresses, and the test actually fails with a change to "for key in address_args.parse_args()". (i don't think we have a corresponding test for querying customers.) address_args.parse_args() actually does not parse the unsupported query parameter, which is the cause for the test failure.  the way it is now, an unsupported query parameter actually just gets ignored because of this. we could make it so the endpoint accesses args through both address_args.parse_args() and request.args.keys(), not sure what we want to do